crosscomponents.cmake

```diff
@@ -7,4 +7,13 @@ set (CLR_CROSS_COMPONENTS_LIST
   sos
   clrjit
   protojit
-)  
+)
+
+if(CLR_CMAKE_PLATFORM_LINUX AND CLR_CMAKE_TARGET_ARCH_ARM)
```

Comment:
Can you please add just what's needed instead of adding and then removing stuff? And why do we actually need to restrict the list for ARM Linux?